placideirandora/epic-mail
# #164148421 built sign in page

#### What does This PR do?
It creates a sign in page.

#### Description of Task to be completed?
1. A sign in link is added to the get started button. 
2. When the button is clicked on, it opens the sign in page.
3. The sign in page allows a user to sign into his/her account.

### How should this be manually tested?
Navigate to the sign in page and check if you get two fields for email and password and a button for signing in.
